Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/Booking.hs

```diff
@@ -67,8 +67,8 @@ data Booking = Booking
     bapCountry :: Maybe Context.Country,
     startTime :: UTCTime,
     riderId :: Maybe (Id DRD.RiderDetails),
-    fromLocation :: DLoc.BookingLocation,
-    toLocation :: DLoc.BookingLocation,
+    fromLocation :: DLocation.Location,
+    toLocation :: [DLocation.Location],
```

Comment:
My opinion, we can use `NonEmpty` here, if we know that list will always contain at least one drop location
It can be implemented as separate refactor PR